README.md

```diff
@@ -65,7 +65,81 @@ A [tenant repository](https://github.com/fluxcd/flux2-multi-tenancy/tree/dev-tea
     └── podinfo-values.yaml
 ```
 
-## Onboard tenants
+
+## Bootstrap the staging cluster
+
+Install the Flux CLI and fork this repository on your personal GitHub account
+and export your GitHub username and repo name:
+
+```sh
+export GITHUB_USER=<your-username>
+export GITHUB_REPO=<repository-name>
+```
+
+Verify that your staging cluster satisfies the prerequisites with:
+
+```sh
+flux check --pre
+```
+
+Set the `--context` argument to the kubectl context to your staging cluster and bootstrap Flux:
+
+```sh
+flux bootstrap github \
+    --context=your-staging-context \
+    --owner=${GITHUB_USER} \
+    --repository=${GITHUB_REPO} \
+    --branch=main \
+    --personal \
+    --path=clusters/staging
+```
+
+At this point the flux cli will ask you for your GITHUB_TOKEN, also known as [Personal Access Token].
```

Comment:
We should mention here that the PAT does not leave the local machine, it's used exclusively by the CLI while the controllers use the generated deploy key.